CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project(unified-memory-framework VERSION 0.1.0 LANGUAGES C)
 # Build Options
 option(UMF_BUILD_SHARED_LIBRARY "Build UMF as shared library" OFF)
 option(UMF_BUILD_OS_MEMORY_PROVIDER "Build OS memory provider" ON)
+option(UMF_BUILD_LIBUMF_PROVIDER_LEVEL_ZERO "Build Level Zero memory provider" OFF)
```

Comment:
if L0 provider implementation has no build-time dependencies why not ON it by default?

Reply:
no, ze_api.h is still used in src/provider/provider_level_zero.c

Reply:
Ok, there is a build-time dependency. Another question regarding distribution, since L0 provider implementation has no runtime dependency to L0 driver we can make L0 provider implementation to be part of core UMF library.

Reply:
I will make this option enabled by default (ON) + check for build deps in cmake + message to user